pysilcam/postprocess.py

```diff
@@ -447,7 +490,7 @@ def montage_maker(roifiles, roidir, pixel_size, msize=2048, brightness=255,
 
 def make_montage(stats_csv_file, pixel_size, roidir,
         auto_scaler=500, msize=1024, maxlength=100000,
-        oilgas=outputPartType.all):
+        oilgas=outputPartType.all, crop_stats_bounds=None):
```

Comment:
E128: continuation line under-indented for visual indent  <!-- Automatically posted by Lintly -->